sycl/doc/GettingStartedAlveo.md

```diff
@@ -0,0 +1,449 @@
+execution mode and is used by the SYCL compiler and XRT runtime.
+
+So to run an example, for example start with
+```bash
+cd $SYCL_HOME/sycl/test/xocc_tests/simple_tests
+```
+- with software emulation:
+  ```bash
+  export XCL_EMULATION_MODE=sw_emu
+  # Configure the simulation environment
+  emconfigutil -f $XILINX_PLATFORM --nd 1 --save-temps
+  # Compile the SYCL program down to a host fat binary including device code for CPU
+  $SYCL_BIN_DIR/clang++ -std=c++2a -fsycl -fsycl-targets=fpga64-xilinx-unknown-sycldevice \
+    parallel_for_ND_range.cpp -o parallel_for_ND_range \
+    -lOpenCL -I/opt/xilinx/xrt/include
+  # Run the software emulation
+  ./parallel_for_ND_range
+  ```
+- with hardware emulation:
+  ```bash
+  export XCL_EMULATION_MODE=hw_emu
+  # Configure the simulation environment
+  emconfigutil -f $XILINX_PLATFORM --nd 1 --save-temps
+  # Compile the SYCL program down to a host fat binary including the RTL for simulation
+  $SYCL_BIN_DIR/clang++ -std=c++2a -fsycl -fsycl-targets=fpga64-xilinx-unknown-sycldevice \
+    parallel_for_ND_range.cpp -o parallel_for_ND_range \
+    -lOpenCL -I/opt/xilinx/xrt/include
+  # Run the hardware emulation
+  ./parallel_for_ND_range
+  ```
+- with real hardware execution on FPGA:
+  ```bash
+  # Instruct only the compiler about hardware execution (\todo: change this API)
+  export XCL_EMULATION_MODE=hw
+  # Compile the SYCL program down to a host fat binary including the FPGA bitstream
+  $SYCL_BIN_DIR/clang++ -std=c++2a -fsycl -fsycl-targets=fpga64-xilinx-unknown-sycldevice \
+    parallel_for_ND_range.cpp -o parallel_for_ND_range \
+    -lOpenCL -I/opt/xilinx/xrt/include
+  # Unset the variable at execution time to have real execution
+  unset XCL_EMULATION_MODE
+  # Run on the real FPGA board
+  ./parallel_for_ND_range
+  ```
+Note that the compilation line does not change, just the environment
+variable and the use or not of `emconfigutil`.
+
+### Running bigger example on real FPGA
+
+To run a SYCL translation of
+https://github.com/Xilinx/SDAccel_Examples/tree/master/vision/edge_detection
+```bash
+cd $SYCL_HOME/sycl/test/xocc_tests/sdaccel_ports/vision/edge_detection
+export XCL_EMULATION_MODE=hw
+$SYCL_BIN_DIR/clang++ -std=c++2a -fsycl \
+    -fsycl-targets=fpga64-xilinx-unknown-sycldevice edge_detection.cpp \
+    -o edge_detection -lOpenCL `pkg-config --libs opencv` \
+    -I/opt/xilinx/xrt/include
+unset XCL_EMULATION_MODE
+# Execute on one of the images
+./edge_detection data/input/eiffel.bmp
+```
+and then look at the `input.bmp` and `output.bmp` images.
+
+
+## Cleaning up some buffer allocation
+
+The XRT memory model is quite richer (and more complex...) than the
+OpenCL memory model: buffers can be allocated on some DDR or HBM
+memory banks, buffers can be shared between different processes on the
+host, etc.
+
+This means that the buffer lifetime is actually handled by the `xocl`
+kernel driver across the machine to manage this memory sharing, if
```

Comment:
"This means that the buffer lifetime is actually handled by the `xocl`
kernel driver across the machine to manage this memory sharing, if
required.",

I think this needs rewording or I at least don't quite grasp it, I lose you at:

"across the machine to manage this memory sharing, if required."